google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableCloudMonitoringExporter.java

```diff
@@ -94,23 +98,28 @@ public static BigtableCloudMonitoringExporter create(
     // TODO: createServiceTimeSeries needs special handling if the request failed. Leaving
     // it as not retried for now.
     settingsBuilder.createServiceTimeSeriesSettings().setSimpleTimeoutNoRetries(timeout);
+
+    MonitoredResource gceOrGkeResource = BigtableExporterUtils.detectResource();
+
+    logger.log(Level.INFO, "Detected resource: " + gceOrGkeResource);
```

Comment:
Can we tone this down to debug logging? Logging the failure to detect resource as a warning makes sense. But dumping it every time the client starts is not something that a customer would wanna see

Reply:
good catch.. this was from me debugging .. will remove it..